api/resource.rb

```diff
@@ -26,6 +26,7 @@ module Properties
       attr_reader :description
       attr_reader :kind
       attr_reader :base_url
+      attr_reader :update_url
```

Comment:
Can you add a comment here explaining what this is for?  And add it to the spreadsheet of resource attributes?

Reply:
Done and done